.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.py]
+indent_size = 4
+
+[*.scala]
+indent_size = 2
```

Comment:
I saw there are differences in the indentation of Scala files and Python, this allow most editors (sublime, atom,...) to adapt automatically